test/test_operators_sparse.jl

```diff
@@ -117,6 +117,21 @@ IEye = identityoperator(b_l)
 @test sparse(IEye) == I
 Icomp = identityoperator(b1a) ⊗ identityoperator(b2a) ⊗ identityoperator(b3a)
 @test IEye == Icomp
+for IEye in (IEye, -IEye, IEye, dagger(IEye))
```

Comment:
IEye is tested twice here?

Reply:
yup, it was supposed to be an adjoint, will fix